source/javascripts/services/_appService.js.erb

```diff
@@ -424,11 +424,11 @@
 					}
 
 					appService.validationWarnings = [];
-					if (validationData.warnings.config !== null) {
+					if (validationData.warnings.config.length != null) {
```

Comment:
The previous version checks if the config isn't null and now there is no null check and we check if the length isn't null? The length of an array won't be null.

Reply:
nope, fixed